README.md

```diff
@@ -1,9 +1,14 @@
 # ![ChRIS logo](https://github.com/FNNDSC/ChRIS_ultron_backEnd/blob/master/docs/assets/logo_chris.png) ChRIS_ui
 
-This repository contains the reference UI for ChRIS, allowing users to create and interact with dynamic containerized workflows.
+This repository contains the reference UI for ChRIS, allowing users to create and interact with dynamic containerized workflows. The ChRIS UI is written primarily in [TypeScript](https://www.typescriptlang.org/) and [React](https://reactjs.org/), and uses the [PatternFly](https://github.com/patternfly/patternfly) React pattern library.
+
+![Homepage](src/assets/images/home.png?raw=true)
```

Comment:
I would suggest moving the screenshot out of the `src/assets` folder, because it is not part of the actual source code/app's assets.

Actually, maybe use one of these urls instead:

- https://github.com/FNNDSC/CHRIS_docs/blob/master/images/mpc/Feed-Detail-Screencapture-PACS-selected.png
- https://chrisproject.org/assets/images/chris-screenshot.jpg